hollow/src/main/java/com/netflix/hollow/core/index/HollowPrimaryKeyIndex.java

```diff
@@ -506,6 +503,27 @@ private void deltaUpdate(int hashTableSize, int bitsPerElement) {
         memoryRecycler.swap();
     }
 
+    private int findOrdinalBucket(int bitsPerElement, FixedLengthElementArray hashedArray, int hashCode, int hashMask, int prevOrdinal) {
+        int startBucket = hashCode & hashMask;
+        int bucket = startBucket;
+        long value;
+        do {
+            value = hashedArray.getElementValue((long)bucket * (long)bitsPerElement, bitsPerElement);
+            if (prevOrdinal + 1 == value) {
+                return bucket;
+            }
+            bucket = (bucket + 1) & hashMask;
+        } while (value != 0 && bucket != startBucket);
```

Comment:
this is meant to be zero right, not ORDINAL_NONE?

Reply:
Yes, it's meant to be zero. A zero `long` value in the hash map corresponds to an empty ordinal.  
e.g. this is code further down when adding new ordinals to find an empty bucket:
```
            while(hashedArray.getElementValue((long)bucket * (long)bitsPerElement, bitsPerElement) != 0)
                bucket = (bucket + 1) & hashMask;
```
(Note although this lacks a bound check for wrapping around the hash table is sized such that it is guaranteed that empty buckets will exist for the cardinality, so the loop will terminate.)